test/Analysis/Inputs/xtu-other.cpp

```diff
@@ -75,3 +75,10 @@ typedef struct AVBuffer avt;
 int avtSize(void){
 	return sizeof(avt);
 }
+
+template <typename T> class X { T t; };
+
+int fun_with_unsupported_node(){
+  using typeAliasDecl = X<int>;
```

Comment:
We plan to support this feature really soon.  Introducing that support will break this test. Is there a more esoteric and rare node that we do not plan to support anytime soon or at all?